arch/univariate/mean.py

```diff
@@ -1917,7 +1917,8 @@ def arch_model(
     am: ARCHModel
 
     known_mean = ("zero", "constant", "harx", "har", "ar", "arx", "ls")
-    known_vol = ("arch", "figarch", "garch", "harch", "constant", "egarch")
+    known_vol = ("arch", "figarch", "garch", "harch", "constant", "egarch",
```

Comment:
Please use black to format the file

Reply:
hi I used black to format the file, imported FIGARCH and opened a PR for adding APARCH in the smoke test. I hope I did well, it's my first PR